src/cpx_genericcallbacks.jl

```diff
@@ -0,0 +1,260 @@
+@enum CbSolStrat CPXCALLBACKSOLUTION_CHECKFEAS CPXCALLBACKSOLUTION_PROPAGATE
```

Comment:
@odow @mlubin do these names (and those below for CbInfo) need to be so verbose, with the `CPXCALLBACK` prefix on all of them?

Reply:
I think they're fine for now.